pages/windows/choco.md

```diff
@@ -1,7 +1,7 @@
 # choco
 
 > The Chocolatey package manager for Windows.
-> Some subcommands such as `install`, `upgrade`, `uninstall` have their own usage documentation.
+> Some subcommands such as `new`, `pack`, `pin` have their own usage documentation.
```

Comment:
Make it so that the commands listed here are part familiar, part unfamiliar. Put `install` and `upgrade` as the first two and put one of these three as the third example.